a1-gillian-and-sharon.py

```diff
@@ -0,0 +1,122 @@
+#
+# inf1340, section L102
+# assignment 1 - due 2018-10-17
+# Chen, Gillian Siyuan
+# Lam, Sharon
+#
+# POS program for MinMax
+# v 1.0 - 2018-10-09
+# compatible with Python version 3.6.5
+# source file: a1-gillian-and-sharon.py)
+#
+
+""" Test cases:
+
+"""
+
+PRICE_SINGLE = 1.00
+PRICE_SMALL = 5.00
+PRICE_LARGE = 19.00
+UPC_SINGLE = 111111
+UPC_SMALL = 666666
+UPC_LARGE = 242424
+HST_RATE = 0.13
+
+# display a welcome message
+def display_welcome():
+    print("Welcome to MinMax! The cashier will scan the items you wish to purchase.")
+
+# prompt cashier to scan for each item, show the running total, and show the total bill
+def get_barcode():
+    scanning = True
+    running_total = 0
+    number_of_singles = 0
+    number_of_smalls = 0
+    number_of_larges = 0
+
+    while (scanning):
+        cashier_input = input("Please scan an item: ")
+        cashier_input = int(cashier_input)
+
+        if cashier_input == 0:
+            print("Scanning complete.")
+            scanning = False
+        elif cashier_input == UPC_SINGLE:
+            cost = PRICE_SINGLE + (PRICE_SINGLE * HST_RATE)
+            cost_rounded = round((round(cost / 0.05) * 0.05), 2)
```

Comment:
repetitive code that can be placed at the end of the while loop (so that you don't have to repeat the same code 3 times)